tsconfig.build.json

```diff
@@ -1,13 +1,9 @@
 {
+  "extends": "@tsconfig/node18",
   "compilerOptions": {
-    "module": "commonjs",
-    "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "target": "es6",
-    "noImplicitAny": true,
-    "moduleResolution": "node",
-    "sourceMap": true,
-    "declaration": true,
```

Comment:
We need to keep `declaration: true`, otherwise the build doesn't contain type information and TypeScript will complain :/

Reply:
Testing this now with `1.8.0-next.1`